core/tests/protractor/learnerDashboard.js

```diff
@@ -242,9 +246,9 @@ describe('Learner dashboard functionality', function() {
       'MultipleChoiceInput', 'Those were all the questions I had!');
     player.submitAnswer('Continue', null);
     player.submitFeedback(feedback);
-    browser.get(general.LEARNER_DASHBOARD_URL);
+    learnerDashboardPage.get();
     browser.waitForAngular();
-    element(by.css('.protractor-test-feedback-section')).click();
+    learnerDashboardPage.navigateToFeedbackSection();
     browser.waitForAngular();
     expect(element.all(by.css(
```

Comment:
This also seems like something that should be part of the LearnerDashboardPage object?

Basically, anything that involves fiddling around with elements should be part of the page object, so that the protractor test itself doesn't have to mess around with the details.

Reply:
Yes, I added that in the learnerDashboardPage object already. Is there something else?

Reply:
I don't think it's done, or rather, if it is, then this line should be learnerDashboardPage.something(...). See previous comment.

Reply:
Yes, saw it, will add that.

Reply:
Done